src/Reveal.Sdk.Dom/Visualizations/KpiTargetVisualization.cs

```diff
@@ -6,10 +6,24 @@
 
 namespace Reveal.Sdk.Dom.Visualizations
 {
+    /// <summary>
+    /// The KPI (key performance indicator) vs target visualization is used for measuring performance against a target through a prior time period.

```

Comment:
you can remove the content within the parenthesis. Customers know what a KPI stands for at this point